tools/src/commands/CommentatorCommand.ts

```diff
@@ -0,0 +1,62 @@
+import { Command } from '@expo/commander';
+import chalk from 'chalk';
+import { link } from '../Formatter';
+
+import { commentOnIssueAsync } from '../GitHubActions';
+import logger from '../Logger';
+
+type ActionOptions = {
+  payload: string;
+};
+
+export type CommentatorComment = {
+  issue: number;
+  body: string;
+};
+
+export type CommentatorPayload = CommentatorComment[];
+
+export default (program: Command) => {
+  program
+    .command('commentator')
+    .alias('comment')
+    .option(
+      '-p, --payload <payload>',
+      'Serialized and escaped JSON describing what and where to comment.'
+    )
+    .asyncAction(main);
+};
+
+async function main(options: ActionOptions) {
+  const payload = parsePayload(options.payload);
+  const commentedIssues: number[] = [];
+
+  if (!Array.isArray(payload)) {
+    throw new Error(`Payload must be an array.`);
+  }
+  for (const comment of payload) {
+    if (!comment.issue || !comment.body) {
+      logger.error('Comment payload is incomplete:', comment);
+      continue;
+    }
+    await commentOnIssueAsync(comment.issue, comment.body);
```

Comment:
Maybe we should be permissive on this one. If somehow the issue id was malformed, it shouldn't block other attempts to create issues, right?